storage/zfs/zfs-tools/pkg.yaml

```diff
@@ -40,6 +40,13 @@ steps:
         # since talos doesn't have a shell, this would fail
         # the default shipped file is a no-op, so safe to remove.
         rm -f /rootfs/usr/local/libexec/zfs/zfs_prepare_disk
+    sbom:
+      outputPath: /rootfs/usr/local/share/spdx/zfs-tools.spdx.json
+      version: {{ .ZFS_DRIVER_VERSION }}
+      cpes:
+        - cpe:2.3:a:openzfs:openzfs:{{ .ZFS_DRIVER_VERSION }}:*:*:*:*:*:*:*
```

Comment:
Don't we already have the same entry in the kernel module build in pkgs? Maybe this shouldn't be duplicated then, but not so sure

Reply:
This also ships `zfs` CLI in addition to kernel module

Reply:
Yes, but should it ship another SBOM with the same data? How would scanners see that potentially?

Reply:
idk, we can leave zfs SBOM here, as it's more than just a kernel module

Reply:
i think we should wrap the one from zfs kernel module and the one from zfs cli in the extension sbom, we can fix once bldr has support, I'll take a look into bldr